Dockerfile

```diff
@@ -26,8 +21,6 @@ ENV PYTHONPATH=/app
 WORKDIR /repo
 COPY entrypoint.sh ../entrypoint.sh
 
-# USER vdbnonroot  # Enable again after removing git from the src code
```

Comment:
This action writes files to the host filesystem (`.mpyl/`, etc) so it cannot run as `vdbnonroot`. We can likely configure the action to run with a specific user, but let's leave that for a follow-up PR.

Reply:
Ah you left this comment while I was reviewing, I'll resolve my question below, hehe.